File: Codeforces/ICPCAssiut/Loop/onePrime/onePrime.py
n = int(input())
count = 0

if n <= 1:
    print("NO")
else:
    for i in range(1, n+1):
        if n % i == 0:
            count += 1

    if count > 2:
        print("NO")
    else:
        print("YES") 

File: Codeforces/ICPCAssiut/Loop/onePrime/OnePrime.java
import java.util.Scanner;

public class OnePrime {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int n, i, count = 0;
        n = s.nextInt();
        if (n <= 1) {
            System.out.println("NO");
        } else {
            for (i = 1; i <= n; i++) {
                if(n % i == 0){
                count++;
                }
            }
        }
        if(count > 2){
            System.out.println("NO");
        }else{
            System.out.println("YES");
        }
    }
}